src/main/java/org/folio/inventory/resources/Instances.java

```diff
@@ -45,22 +35,38 @@
 import org.folio.inventory.support.http.server.ServerErrorResponse;
 import org.folio.inventory.validation.InstancePrecedingSucceedingTitleValidators;
 import org.folio.inventory.validation.InstancesValidators;
-import org.folio.inventory.exceptions.UnprocessableEntityException;
 import org.folio.rest.client.SourceStorageRecordsClient;
 
-import io.vertx.core.http.HttpClient;
-import io.vertx.core.json.Json;
-import io.vertx.core.json.JsonArray;
-import io.vertx.core.json.JsonObject;
-import io.vertx.ext.web.Router;
-import io.vertx.ext.web.RoutingContext;
-import io.vertx.ext.web.handler.BodyHandler;
+import java.io.UnsupportedEncodingException;
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Objects;
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.CompletionStage;
+import java.util.stream.Collectors;
+
+import static io.netty.util.internal.StringUtil.COMMA;
+import static java.lang.String.format;
+import static java.util.concurrent.CompletableFuture.completedFuture;
+import static org.folio.inventory.support.CompletableFutures.failedFuture;
+import static org.folio.inventory.support.EndpointFailureHandler.doExceptionally;
+import static org.folio.inventory.support.EndpointFailureHandler.getKnownException;
+import static org.folio.inventory.support.EndpointFailureHandler.handleFailure;
+import static org.folio.inventory.support.http.server.SuccessResponse.noContent;
+import static org.folio.inventory.validation.InstancesValidators.refuseWhenHridChanged;
 
 public class Instances extends AbstractInstances {
   private static final String INSTANCES_CONTEXT_PATH = INSTANCES_PATH + "/context";
-  private static final String BLOCKED_FIELDS_CONFIG_PATH = INVENTORY_PATH + "/config/instances/blocked-fields";
-  private static final String BLOCKED_FIELDS_UPDATE_ERROR_MESSAGE = "Instance is controlled by MARC record, "
+  private static final String INSTANCE_BLOCKED_FIELDS_CONFIG_PATH = INVENTORY_PATH + "/config/instances/blocked-fields";
+  private static final String INSTANCE_BLOCKED_FIELDS_UPDATE_ERROR_MESSAGE = "Instance is controlled by MARC record, "
     + "these fields are blocked and can not be updated: ";
+  private static final String HOLDINGS_BLOCKED_FIELDS_CONFIG_PATH = INVENTORY_PATH + "/config/holdings/blocked-fields";
```

Comment:
It is strange to have holdings config in `Instances` class. Also, I don't see where these fields for holdings are really blocked the holdings update like it is done for instances: 
![image](https://user-images.githubusercontent.com/5904625/135266189-f8220aee-9de7-42bd-90bb-e55ab45035fb.png)

Reply:
Will do